ui/raidboss/data/07-dt/raid/r5s.ts

```diff
@@ -21,23 +21,29 @@ const snapTwistIdMap: { [id: string]: [SnapCount, EastWest] } = {
   'A728': ['two', 'west'],
   'A729': ['two', 'west'],
   'A72A': ['two', 'west'],
+  'A4DB': ['two', 'west'],
   'A72B': ['two', 'east'],
   'A72C': ['two', 'east'],
   'A72D': ['two', 'east'],
+  'A4DC': ['two', 'east'],
   // 3-snap Twist & Drop the Needle
   'A730': ['three', 'west'],
   'A731': ['three', 'west'],
   'A732': ['three', 'west'],
+  'A4DE': ['three', 'west'],
   'A733': ['three', 'east'],
   'A734': ['three', 'east'],
   'A735': ['three', 'east'],
+  'A4DD': ['three', 'east'],
```

Comment:
Do you have video to verify `A4DE/A4DD`? Matching my recordings against my logs, it looks like you might have reversed these two, although I don't have enough of a sample size to be sure.

Reply:
As far as I have been able to tell, you are correct. I don't think I have a recording of this ID myself, but it does match that lower numbers are the west-safe cleave.

Reply:
Okay. Let me see whether either shows up from tonight's reclears. If not, let's solicit logs from someone else since it's better to have nothing than to be incorrect.

Reply:
I re-checked logs+vod from last week prog, and I only saw one instance of `A4DD`, and none of `A4DE`.  Didn't see either during reclears tonight.
The original commit did have these reversed, but the latest `129fe86` looks correct, e.g., it should be and now is:
`'A4DD': ['three', 'west'],`

Reply:
Yup, just checked tonight's log and it's correct as it stands now.